index.html

```diff
@@ -27,29 +27,200 @@
 
         <section class="headline">
             <h1><br>I'm Carlos<br>Glad to see<br>you!</h1> 
-            <p>I'm a software developer! I can help you<br>build a product , feature or website Look<br>through some of my work and<br>experience! If you like what you see and<br>have a project you need coded, don’t<br>hestiate to contact me.</p>
+            <p>I'm a software developer! I can help you<br>
+              build a product , feature or website Look<br>
+              through some of my work and<br>
+              experience! If you like what you see and<br>
+              have a project you need coded, don’t<br>
+              hestiate to contact me.
+            </p>
             <h3>LET'S CONNECT</h3>
               <div >
                 <ul class="icons-social" >
-                    <li>
-                      <img src="images/twitter.png" class="social" alt="twitter" />
-                    </li>
-                    <li>
-                      <img src="images/Linkedin.png" class="social" alt="Linkedin" />
-                    </li>
-                    <li>
-                      <img src="images/medium.png" class="social" alt="Medium" />
-                    </li>
-                    <li>
-                      <img src="images/github.png" class="social" alt="Github" />
-                    </li>
-                    <li>
-                      <img src="images/angellist.png" class="social" alt="angellist" />
-                    </li>
-                  </ul>
+                  <li>
+                    <img src="images/github.png" class="social" alt="Github">
+                  </li>
+                  <li>
+                    <img src="images/Linkedin.png" class="social" alt="Linkedin">
+                  </li>                               
+                  <li>
+                    <img src="images/angellist.png" class="social" alt="angellist">
+                  </li>
+                  <li>
+                    <img src="images/twitter.png" class="social" alt="twitter">
+                  </li>
+                  <li>
+                    <img src="images/medium.png" class="social" alt="Medium">
+                  </li>
+                </ul>
               </div>
         </section>
 
+        <section class="work-section">
```

Comment:
- [ ] kindly maintain the white background in the work section as in the figma version